frontend/src/app/course/styles/courseCard.scss

```diff
@@ -0,0 +1,15 @@
+hr {
+    margin-left: -1.25rem;
+    margin-right: -1.25rem;
+}
+
+.card {
+    margin: 8px 8px;
+    cursor: pointer;
+    min-width: 200px;
+    width: 100%;
+}
```

Comment:
The convention for sizing is as follow: 
- margins and paddings should use rem 
- if possible use pre defined margins ( see themes/variables.scss )
- widhts and heights use precentages %